Sources/Views/Validation Form/ValidationFormView.swift

```diff
@@ -0,0 +1,245 @@
+//
+//  ValidationFormView.swift
+//  PovioKit
+//
+//  Created by Toni Kocjan on 10/09/2020.
+//  Copyright © 2020 Povio Labs. All rights reserved.
+//
+
+import UIKit
+
+public protocol ValidationFormViewDelegate: AnyObject {
+  func validationFormDidScroll<F: ValidationForm>(_ validationFormView: ValidationFormView<F>)
+  func validationFormDidTranslateForKeyboardAppereance<F: ValidationForm>(
+    _ validationFormView: ValidationFormView<F>,
+    offset: CGFloat,
+    withDuration: TimeInterval)
+}
+
+public class ValidationFormView<F: ValidationForm>: UIView, UICollectionViewDelegateFlowLayout {
+  private let validationForm: F
+  private lazy var collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+  private let layout = UICollectionViewFlowLayout()
```

Comment:
It's an implementation detail that should be hidden. For instance, we could replace the underlying layout backed to `UITableView`, which would break user code.